package.json

```diff
@@ -71,7 +71,9 @@
     "jsdom": "14.0.0",
     "jsdom-global": "3.0.2",
     "prettier": "1.16.4",
-    "react-test-renderer": "16.8.4",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "16.8.6",
```

Comment:
Shouldn't we have the same version as `react` and `react-dom`? If it's the case, and we're using peer dependencies, don't you think we can't have a fixed version?

Reply:
We don't need `react-test-renderer` because it's only a `devDependencies`